### ISSUE TITLE
Markdown rendering a codeblock using markdown false positives

### ISSUE THREAD

**Issue by marcofranssen** [markdownlint/markdownlint#324]
## Description

When embedding a markdown example in a codeblock MD022 error is shown which is a false positive. Also tried changing it to txt to circumvent the problem.

### Environment

https://github.com/philips-labs/tabia/pull/30/files

### Expected Behavior

Either I should be valid according to the linter or the linter should skip the linting of the code-block itself

### Actual Behavior
MD022 Headers should be surrounded by blank lines

### Markdown example

```md
# Example markdown with embedded markdown

Below an example of Markdown in markdown

    ```md
    # I am getting a MD022 linter error

    This should not be the case as I'm the start of a markdown file.

    Either I should be valid according to the linter or the linter should skip the linting of the code-block itself
    ```

```

### renders as

# Example markdown with embedded markdown

Below an example of Markdown in markdown

```md
# I am getting a MD022 linter error

This should not be the case as I'm the start of a markdown file.

Either I should be valid according to the linter or the linter should skip the linting of the code-block itself
```

**Comment by HariSekhon** [comment]
+1

**Comment by jaymzh** [comment]
This is fixed on the main branch
